responsibleai/requirements.txt

```diff
@@ -1,7 +1,7 @@
 dice-ml>=0.6.1,<0.7
 econml>=0.12.0b1
 erroranalysis>=0.1.4
-interpret-community>=0.17.0
+interpret-community>=0.18.0
```

Comment:
This should be 0.18.1? That has the sparse fix from Ilya.

Reply:
Sure, can do